MeDrop/MeDrop/Sources/Creation/View/CreationMainDummyView.swift

```diff
@@ -12,10 +12,15 @@ struct CreationMainDummyView: View {
     @StateObject var myCards = EnvironmentData()
     @StateObject var yourCards = EnvironmentData()
     
+    @State var showingCreation = false
```

Comment:
```suggestion
    @State private var isShowingCreation = false
```
이 struct에서만 사용된다면 접근제한자 설정으로 캡슐화를 시키면 어떨까요 ? 🤩

Reply:
넵! 좋습니다